styles/globals.scss

```diff
@@ -1,16 +1,75 @@
-html,
-body {
-  padding: 0;
-  margin: 0;
+@import 'bootstrap/dist/css/bootstrap.min.css';
+
```

Comment:
I inline commented all of the settings I didn't find intuitive, but flag any I missed!